### ISSUE TITLE
Documentation of retained metadata for `read_sas`, `read_xpt`, `write_xpt`

### ISSUE THREAD

**Issue by kaz462** [tidyverse/haven#719]
Would it be possible to document a list of retained metadata when using `read_sas`, `read_xpt`, `write_xpt`? to help users understand which metadata is kept through read, as well as the attributes that can be customized during write. Thank you!

**Comment by botsp** [comment]
Yes, hope it can keep the Sort Information when sas dataset is imported into R;
